README.md

```diff
@@ -339,7 +339,7 @@ export FORGIT_LOG_FZF_OPTS='
 
 # 📦 Optional dependencies
 
-- [`delta`](https://github.com/dandavison/delta) / [`diff-so-fancy`](https://github.com/so-fancy/diff-so-fancy): For better human readable diffs.
+- [`delta`](https://github.com/dandavison/delta) / [`diff-so-fancy`](https://github.com/so-fancy/diff-so-fancy): For better human-readable diffs.
```

Comment:
Same here. Would prefer to have this in a follow up PR with the change I've commented on above.